proto/viam/app/v1/app.proto

```diff
@@ -1018,3 +1021,11 @@ message ListKeysRequest {
 message ListKeysResponse {
   repeated APIKeyWithAuthorizations api_keys = 1;
 }
+
+message RotateKeyRequest {
+  string id = 1;
+}
+
+message RotateKeyResponse {
+  string id = 1;
+}
```

Comment:
oh sorry i wasn't super clear in my original comment but i DO think we want to include `key` in the response. since we're generating a new key, we'll want to send that along to display. although i haven't fully though through the how the client side will be calling this. do you think the UI will make this rotate key request and replace what's shown in the api keys table? or do you think the UI should recall `ListKeys` after rotating?